### PR TITLE
dos2unix, mac2unix, unix2dos, unix2mac: fix error in descriptions

### PR DESCRIPTION
**Pull request by xphade**
Unix systems use the Line Feed character (LF) to specify a newline while
early macOS versions used a Carriage Return (CR). In the descriptions of
`dos2unix` and all related commands, this was reported the wrong way
round.

I fixed the error for all languages. I can speak English and Spanish but not Chinese, so I used Google Translate for that. Maybe someone speaking Chinese can verify that it is in the correct order now.

- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 7.4.2

### CODE REVIEW

**Comment by TimStricker** [comment]
Hey everyone, since there has been no feedback for a couple of weeks: Do you have any questions about this? Or is there anything missing from my side?

**Review by Waples — APPROVED** [review]

**Comment by Waples** [comment]
Hi Tim, lgtm, pending another reviewer

**Pull request merged**
